content/open-source/octo-sts/trust-policy-reference.md

```diff
@@ -0,0 +1,578 @@
+- Read: View issues
+- Write: Create, update, close issues
+
+**packages** (`read`, `write`)
+- Read: Download packages
+- Write: Publish and delete packages
+
+**pages** (`read`, `write`)
+- Read: View GitHub Pages settings
+- Write: Configure GitHub Pages
+
+**projects** (`read`, `write`, `admin`)
+- Read: View projects
+- Write: Edit projects
+- Admin: Manage project settings
+
+**pull_requests** (`read`, `write`)
+- Read: View pull requests
+- Write: Create and manage pull requests
+
+**statuses** (`read`, `write`)
+- Read: View commit statuses
+- Write: Create commit statuses
+
+**workflows** (`read`, `write`)
+- Read: View workflow files
+- Write: Update workflow files (critical for tools like Renovate)
+
+### Organization Permissions
+
+**members** (`read`, `write`)
+- Read: View organization members
+- Write: Manage organization membership
+
+**administration** (`read`, `write`)
+- Read: View organization settings
+- Write: Manage organization settings
+
+### Additional Permissions
+
+For the complete and current list of permissions, see the [octo-sts README](https://github.com/octo-sts/app#octo-sts-github-permissions).
+
+## Pattern Matching Examples
+
+### Example 1: GitHub Actions on Main or Develop
+
+```yaml
+issuer: https://token.actions.githubusercontent.com
+subject_pattern: "repo:chainguard-dev/example:ref:refs/heads/(main|develop)"
+
+permissions:
+  contents: read
+  pull_requests: write
+```
+
+### Example 2: Any Google Email in Organization
+
+```yaml
+issuer: https://accounts.google.com
+subject_pattern: "[0-9]+"
+claim_pattern:
+  email: ".*@chainguard.dev"
+  email_verified: "true"
+
+permissions:
+  contents: read
+```
+
+### Example 3: Kubernetes Service Account in Production Namespace
+
+```yaml
+issuer: https://kubernetes.default.svc.cluster.local
+subject_pattern: "system:serviceaccount:production:.*"
+
+permissions:
+  contents: write
+  deployments: write
+```
+
+### Example 4: GitHub Actions from Any Repository in Organization
+
+```yaml
+issuer: https://token.actions.githubusercontent.com
+subject_pattern: "repo:chainguard-dev/.*:ref:refs/heads/main"
+claim_pattern:
+  repository_owner: "chainguard-dev"
+
+permissions:
+  contents: read
+```
+
+## Security Considerations
+
+### Principle of Least Privilege
+
+Grant only the minimum permissions required:
+
+```yaml
+# Good: Specific permissions for specific task
+permissions:
+  contents: read
+  pull_requests: write
+```
+
+```yaml
+# Avoid: Overly broad permissions
+permissions:
+  contents: write
+  issues: write
+  pull_requests: write
+  workflows: write
+  packages: write
+```
+
+### Subject Matching
+
+Prefer exact subject matches over patterns:
+
+```yaml
+# Better: Exact match
+subject: repo:org/repo:ref:refs/heads/main
+```
+
+```yaml
+# Acceptable but broader: Pattern match
+subject_pattern: "repo:org/repo:ref:refs/heads/.*"
+```
+
+### Branch Protection
+
+Even with `contents: write`, enable branch protection to require pull requests:
```

Comment:
This line doesn't scan for me. Is it saying "even with this permission, you should still require it to submit pull requests rather than merging changes without review"?